Automattic/wp-calypso
# Refactor G Suite Features to Own Component

#### Changes proposed in this Pull Request

* Move the "G Suite Features" of the `GSuitePurchaseCta` component to its own separate component so they can be reused in the future.

* Refactor props of `GSuitePurchaseCta` to minimize information that needs to be extracted

* Add `type` prop to G Suite Features to force them to render in a vertical list

**The G Suite Features ( unchanged in appearance ) in the Purchase Call-to-Action**

<img width="1057" alt="screen shot 2019-02-22 at 3 23 22 pm" src="https://user-images.githubusercontent.com/2810519/53276929-30ab6f00-36b6-11e9-82c5-97f3c84ba235.png">



#### Testing instructions

1. Navigate to `/domains/manage/:domain/email/:site` for a custom domain and site with no G Suite on both the branch and WordPress.com/Production.
1. Verify there is no discernible difference between them
1. run `npm run test-client client/my-sites/domains/domain-management/gsuite`
1. verify that all tests pass
